helpdesk/models/partner.py

```diff
@@ -0,0 +1,36 @@
+from odoo import api, fields, models
+
+
+class Partner(models.Model):
+
+    _inherit = "res.partner"
+
+    helpdesk_ticket_ids = fields.One2many(
+        comodel_name="helpdesk.ticket",
+        inverse_name="partner_id",
+        string="Assigned tickets",
+    )
+
+    helpdesk_ticket_count = fields.Integer(
+        compute="_compute_helpdesk_ticket_count", string="Tickets"
+    )
+
+    @api.depends("helpdesk_ticket_ids")
+    def _compute_helpdesk_ticket_count(self):
+        for record in self:
+            helpdesk_ticket_count = len(record.helpdesk_ticket_ids)
```

Comment:
Maybe count only open tickets (state.closed != False) or keep to counts for both amounts and show them both in the button.